### ISSUE TITLE
keyword stacked for bar plots not documented anywhere.

### ISSUE THREAD

**Issue by joooeey** [pandas-dev/pandas#32759]
#### Code Sample, a copy-pastable example if possible

```python
df.plot.bar(stacked=True)
df.plot(kind='bar', stacked=True)
```
#### Problem description

There is a keyword argument to create a stacked bar graph. This kwarg is not documented anywhere in the Pandas documentation.

#### Expected Output

The documentations for the methods `df.plot.bar` and `df.plot` should include a description of the kwd `stacked`

#### Version

Pandas 1.0.1

**Comment by joybhallaa** [comment]
take

**Comment by joooeey** [comment]
This issue was fixed by a keyword description added in PR #33596 and an example added in PR #33337 .
